pkgs/applications/audio/CHOWTapeModel/default.nix

```
@@ -1,62 +1,106 @@
 { alsa-lib
+, at-spi2-core
+, cmake
 , curl
+, dbus
+, epoxy
 , fetchFromGitHub
 , freeglut
 , freetype
+, gtk3
+, lib
 , libGL
 , libXcursor
+, libXdmcp
 , libXext
 , libXinerama
 , libXrandr
+, libXtst
+, libdatrie
 , libjack2
+, libpsl
+, libselinux
+, libsepol
+, libsysprof-capture
+, libthai
+, libxkbcommon
+, pcre
 , pkg-config
 , python3
+, sqlite
 , stdenv
-, lib
+, util-linuxMinimal
+, webkitgtk
+, gcc-unwrapped
 }:
 
 stdenv.mkDerivation rec {
   pname = "CHOWTapeModel";
-  version = "unstable-2020-12-12";
+  version = "2.8.0";
 
   src = fetchFromGitHub {
     owner = "jatinchowdhury18";
     repo = "AnalogTapeModel";
-    rev = "a7cf10c3f790d306ce5743bb731e4bc2c1230d70";
-    sha256 = "09nq8x2dwabncbp039dqm1brzcz55zg9kpxd4p5348xlaz5m4661";
+    rev = "v${version}";
+    sha256 = "0rl04hr6gpqdkjvvvxqv4qiqrwjd0gmc9nlpqmdjdlbgm195jmzv";
     fetchSubmodules = true;
   };
 
   nativeBuildInputs = [
     pkg-config
+    cmake
   ];
 
   buildInputs = [
     alsa-lib
+    at-spi2-core
     curl
+    dbus
+    epoxy
     freeglut
     freetype
+    gtk3
     libGL
     libXcursor
+    libXdmcp
     libXext
     libXinerama
     libXrandr
+    libXtst
+    libdatrie
     libjack2
+    libpsl
+    libselinux
+    libsepol
+    libsysprof-capture
+    libthai
+    libxkbcommon
+    pcre
     python3
+    sqlite
+    # util-linuxMinimal
+    # webkitgtk
+    gcc-unwrapped
   ];
 
-  buildPhase = ''
-    cd Plugin/
-    ./build_linux.sh
-  '';
+  cmakeFlags = [ "-DCMAKE_AR=${gcc-unwrapped}/bin/gcc-ar"
```

```suggestion
  cmakeFlags = [ "-DCMAKE_AR=${stdenv.cc.targetPrefix}ar"
```
same for the two other below

---

When I do this, I get lots of linker errors.
Oddly enough, the build output in that case has all 3 flags twice; once with a sensible path to the store, and once without a path, so like this:
```
DCMAKE_AR=ar
```

When I remove these lines all together, I still get the flags with full path, but also the linker errors.
When I put back the original lines (as in the current state of this PR), it all builds just fine.

---

More detailed info [here](https://github.com/NixOS/nixpkgs/pull/136761#discussion_r764816618).